packages/eth2.0-state-transition/src/util/duties.ts

```diff
@@ -25,8 +25,6 @@ export function getCommitteeAssignment(
   validatorIndex: ValidatorIndex
 ): CommitteeAssignment {
 
-  const nextEpoch = getCurrentEpoch(config, state) + 1;
-  assert(epoch <= nextEpoch);
```

Comment:
why did you remove this?

Reply:
yeah I'm concerning about this too. When I run e2e `dev.test.ts` it failed at this line.
+ Given SLOTS_PER_EPOCH=5
+ At slot 5, the validator asks for duties of next epoch, which is 2. In the above context `epoch=2`
+ But `nextEpoch` is evaluated as 1 because it only see blocks 0,1,2,3,4 => `getCurrentEpoch` evaluated to 0

I don't see anything wrong here so I have to comment this out. Do you think this is because we changed something in validator?

Reply:
Hmm, I think it's okay to bump next epoch = currentEpoch +2 as this is just to protect us from doing way too many epoch transitions if there is malicious user